requirements-dev.txt

```diff
@@ -1,10 +1,10 @@
-pytest==6.2.5
+pytest>=7.0.0
 pytest-mypy
 pytest-isort
-pytest-flake8
+pytest-flake8-v2
```

Comment:
why do we want to switch to flake8-v2?

Reply:
I usually use flake8-v2 as it lets you specify the config in the `pyproject.toml` file within the `[flake8]` section.

However since I didn't change that config, using v2 has no benefit. As v2 currently has issues with the [stacktrace](https://github.com/VRGhost/pytest-flake8/pull/12), I reverted back to the original version.